CONTRIBUTING.md

```diff
@@ -1,5 +1,38 @@
 # How to Contribute
 
-This project is part of Deis. You can find the latest contribution
-guidelines [at the Deis project](https://github.com/deis/deis/blob/master/CONTRIBUTING.md).
+We love getting Pull Requests (PRs). To make sure we keep code quality
+and consistency high, we do have some process in place. In a nutshell:
+
+- Code should follow Go coding standards and pass `go lint` and `go vet`
```

Comment:
We should also say something about these tools being run automatically on pull requests (which we should set up for those projects that don't have PR builds enabled with anything valuable).